pkg/driver/controller.go

```diff
@@ -55,9 +55,10 @@ var (
 	}
 )
 
-const isManagedByDriver = "true"
+const trueStr = "true"
+const isManagedByDriver = trueStr
```

Comment:
Could we remove `trueStr` and simplify this to 
```
const isManagedByDriver = "true"
```

Reply:
`trueStr` is used in other places (where "is managed by driver" doesn't make sense) so I don't think we can - if we want to remove one of the two it should be `isManagedByDriver`

Reply:
I left `isManagedByDriver` in because it was used in quite a few places to enhance readability. Can replace if we want.

I do not think we should be using isManagedByDriver to signify "true" in other places in the file.